strings_lab_assignment.js

```diff
@@ -0,0 +1,78 @@
+// 1. console.log the first character of a string.
```

Comment:
Overall good understanding of string properties and methods. One thing to pay attention while you are coding is to pay attention whether you are hard coding or not. hard coding is a practice of making code that only works with certain input instead of making your code adoptable to differing inputs. Hard coding is a bad practice and we should minimize that. Other than that, keep up the good work and do bonus questions next time.